contributing/TESTING.md

```diff
@@ -268,7 +268,7 @@ If a Percy test flakes and you would like to trigger a rerun, you can't do that
 
 ### Running Visual Diff Tests Locally
 
-You can also run the visual tests locally during development. You must first create a free Percy account at [https://percy.io](https://percy.io), create a project, and set the `PERCY_PROJECT` and `PERCY_TOKEN` environment variables using the unique values you find at `https://percy.io/<org>/<project>/integrations`. Once the environment variables are set up, you can run the AMP visual diff tests as described below.
+You can also run the visual tests locally during development. You must first create a free Percy account at [https://percy.io](https://percy.io), create a project, and set the `PERCY_TOKEN` environment variable using the unique values you find at `https://percy.io/<org>/<project>/integrations`. Once the environment variable is set up, you can run the AMP visual diff tests as described below.
```

Comment:
Nit: Should this be “unique value“ (singular)?

Reply:
Done